steampipeconfig/load_mod_test.go

```diff
@@ -408,3 +409,67 @@ func executeLoadTest(t *testing.T, name string, test loadModTest, wd string) {
 		t.Errorf("Test: '%s'' FAILED : expected:\n\n%s\n\ngot:\n\n%s", name, expectedStr, actualString)
 	}
 }
+
+type loadResourceNamesTest struct {
+	source   string
+	expected interface{}
+}
+
+var testCasesLoadResourceNames map[string]loadResourceNamesTest
+
+func init() {
+	testCasesLoadResourceNames = map[string]loadResourceNamesTest{
+		"test_load_mod_resource_names_workspace": {
+			source: "test_data/mods/test_load_mod_resource_names_workspace",
+			expected: &modconfig.WorkspaceResources{
+				Benchmark: map[string]bool{"benchmark.test_workspace": true},
+				Control:   map[string]bool{"control.test_workspace_1": true, "control.test_workspace_2": true, "control.test_workspace_3": true},
+				Query:     map[string]bool{"query.query_control_1": true, "query.query_control_2": true, "query.query_control_3": true},
+			},
+		},
+	}
+}
+
+func TestLoadModResourceNames(t *testing.T) {
+	for name, test := range testCasesLoadResourceNames {
+
+		modPath, _ := filepath.Abs(test.source)
+		names, err := LoadModResourceNames(modPath, loadWorkspaceOptions)
+
+		if err != nil {
+			if test.expected != "ERROR" {
+				t.Errorf("Test: '%s'' FAILED with unexpected error: %v", name, err)
+			}
+			continue
+		}
+
+		if test.expected == "ERROR" {
+			t.Errorf("Test: '%s'' FAILED - expected error", name)
+			continue
+		}
+
+		// to compare the benchmarks
+		benchmark_expected := test.expected.(*modconfig.WorkspaceResources).Benchmark
+		flag_b := reflect.DeepEqual(names.Benchmark, benchmark_expected)
```

Comment:
no need for flag, you can just uses 
```
if reflect.DeepEqual(names.Benchmark, benchmark_expected){
}
```

Reply:
Made the change in e16f0936849f0f5531d2ad7b686d66daf2026cd3